packages/puppeteer-core/src/bidi/Input.ts

```diff
@@ -721,57 +720,28 @@ class BidiTouch implements TouchHandle {
  */
 export class BidiTouchscreen extends Touchscreen {
   #page: BidiPage;
-  #idGenerator = createIncrementalIdGenerator();
-  #touches: BidiTouch[] = [];
+  declare touches: BidiTouchHandle[];
 
   constructor(page: BidiPage) {
     super();
     this.#page = page;
   }
 
-  removeHandle(handle: BidiTouch): void {
-    const index = this.#touches.indexOf(handle);
-    if (index === -1) {
-      return;
-    }
-    this.#touches.splice(index, 1);
-  }
-
   override async touchStart(
     x: number,
     y: number,
     options: BidiTouchMoveOptions = {},
   ): Promise<TouchHandle> {
-    const id = this.#idGenerator();
+    const id = this.idGenerator();
     const properties: Bidi.Input.PointerCommonProperties = {
       width: 0.5 * 2, // 2 times default touch radius.
       height: 0.5 * 2, // 2 times default touch radius.
       pressure: 0.5,
       altitudeAngle: Math.PI / 2,
     };
-    const touch = new BidiTouch(this.#page, this, id, x, y, properties);
+    const touch = new BidiTouchHandle(this.#page, this, id, x, y, properties);
     await touch.start(options);
-    this.#touches.push(touch);
+    this.touches.push(touch);
     return touch;
   }
-
-  override async touchMove(
-    x: number,
-    y: number,
-    options: BidiTouchMoveOptions = {},
```

Comment:
but @Lightning00Blade I thought you wanted to keep the `options` parameter?

Reply:
Sorry for the confusion, I didn't understand its the reason for not extracting the logic. 
Considering that we don't test or expose it, I have opted-out of it.

Reply:
Thanks for the clarification. That was indeed the reason. So #13254 could get rid of it in the other place as well :)